info.json

```diff
@@ -0,0 +1,615 @@
+                    "editable": true,
+                    "visible": true,
+                    "type": "text",
+                    "name": "filter.guid",
+                    "value": "",
+                    "description": "Filter by the The globally unique sensor identifier"
+                },
+                {
+                    "title": "External IP Address",
+                    "required": false,
+                    "editable": true,
+                    "visible": true,
+                    "type": "text",
+                    "name": "filter.externalIpAddress",
+                    "value": "",
+                    "description": "Filter by the WAN IP address of the machine"
+                },
+                {
+                    "title": "Internal IP Address",
+                    "required": false,
+                    "editable": true,
+                    "visible": true,
+                    "type": "text",
+                    "name": "filter.internalIpAddress",
+                    "value": "",
+                    "description": "Filter by the LAN IP address of the machine"
+                },
+                                {
+                    "title": "Custom Tags",
+                    "required": false,
+                    "editable": true,
+                    "visible": true,
+                    "type": "text",
+                    "name": "filter.customTags",
+                    "value": "",
+                    "description": "Filter by a list of custom sensor tags assigned to the machine"
+                },
+                {
+                    "title": "Department",
+                    "required": false,
+                    "editable": true,
+                    "visible": true,
+                    "type": "text",
+                    "name": "filter.department",
+                    "value": "",
+                    "description": "Filter by the value assigned to the machine for the DEPARTMENT sensor tag"
+                },
+                {
+                    "title": "Custom Filters",
+                    "required": false,
+                    "editable": true,
+                    "visible": true,
+                    "type": "text",
+                    "name": "raw_filters",
+                    "placeholder": "{\"fieldName\":\"osType\",\"operator\":\"Equals\",\"values\":[\"WINDOWS\",\"LINUX\"]}",
+                    "description": "A comma separated list of filters, check documentation at https://nest.cybereason.com/documentation/api-documentation/all-versions/query-sensors"
+                }
+            ]
+        },
+        {
+            "operation": "get_malops",
+            "title": "Get Incidents",
+            "description": "Returns details about all Auto Hunt Malops and Endpoint Protection Malops (version:20.1.43+)",
+            "category": "investigation",
+            "annotation": "get_malops",
+            "output_schema":
+            {},
```

Comment:
possible to add schema? helps in using the output in subsequent steps in workflows